CONTRIBUTING.md

```diff
@@ -1,26 +1,26 @@
-# How to Contribute
```

Comment:
In the beginning, I edited this file to use another markdown "convention". Seems stupid now, so changed it back to how ember-cli generates it.